src/main/java/org/jabref/gui/importer/GrobidOptInDialogHelper.java

```diff
@@ -23,14 +23,15 @@ public static boolean showAndWaitIfUserIsUndecided(DialogService dialogService,
             return true;
         }
         if (preferences.isGrobidOptOut()) {
-            return false;
+            return preferences.isGrobidEnabled();
```

Comment:
Can we not completely remove the `isGrobidOptOut` check and everything else related to the opt-out?

Reply:
Hi I've removed all the opt out stuff from the dialoghelper. Could you have another look? The only thing I'm confused about though is that the dialog is meant to be single instance. Wouldn't removing opt out cause it to show the box everytime import if Grobid is disabled in prefs?